docs/source/admin/fs/ssh.rst

```diff
@@ -93,3 +93,18 @@ To specify the drive, you need to use the following format:
      username: "username"
      password: "password"
      protocol: "ssh"
+
+Windows shared folders
+~~~~~~~~~~~~~~
```

Comment:
We need to add the same exact number of characters as the title here. So it should be `~~~~~~~~~~~~~~~~~~~~~~`.